.travis.yml

```diff
@@ -9,5 +9,5 @@ python: 3.5
 install:
   - python setup.py develop
   - pip install coveralls
-script: coverage run -m unittest discover tests/$TEST_SUITE
+script: travis_wait coverage run -m pytest tests/$TEST_SUITE
```

Comment:
[20min](https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received) sounds like not enough as tests CI failed…